bin/dev/install_tools.sh

```diff
@@ -1,90 +1,54 @@
 #!/bin/bash
 set -o errexit -o nounset
 
-# Get version information
-. "$(dirname "$0")/versions.sh"
+# Get version information and set destination paths
+. "$(dirname "$0")/../common/versions.sh"
+
+if id -u vagrant >& /dev/null; then
+  SCF_BIN_DIR="${SCF_BIN_DIR:-/usr/local/bin}"
```

Comment:
Can't we just set this in the Vagrantfile where we call this? Then this can stay as a simple `SCF_BIN_DIR=${SCF_BIN_DIR:-output/bin}` and everything should be happy.